tests/lint/lintInstall/requirements.txt

```
@@ -0,0 +1,5 @@
+####### requirements.txt #######
+#
+###### Requirements for automated lint ######
+flake8 == 3.7.7

```

3.7.8 is now out. May be go for the most recent version while at it. I understand that we couldn't keep on changing the version.

See also https://pip.pypa.io/en/stable/reference/pip_install/#id29

```suggestion
flake8 ~= 3.7.7
```

This ensures a compatible release is installed.

---

> This ensures a compatible release is installed.

The intention is a compatible release, it relies on the judgement of the human setting the version number. That said, a breakage here is likely to be fairly low impact for us, so I think this makes sense.